components/Profile/requests.js

```diff
@@ -38,3 +38,20 @@ export const stake = async ({ account, socialId, serviceId, stakingInstance }) =
     throw error;
   }
 };
+
+export const approveServiceTransfer = async ({ account, serviceId, address }) => {
+  try {
+    const contract = getServiceRegistryL2Contract();
+    const approveFn = contract.methods.approve(address, serviceId);
```

Comment:
@kupermind the transfer approve method

Reply:
I'd change `address` to something more meaningful, like `contractAddress` or `managerAddress`, because it can be confused with just the `account`.

Reply:
updated @kupermind